src/connectors/h1_collect.py

```diff
@@ -0,0 +1,316 @@
+
+    timestamp = datetime.utcnow()
+    api_identifier = options['api_identifier']
+    api_token = options['api_token']
+    account_id = options['account_id']
+    program_name = options['program_name']
+
+    if ingest_type == 'transaction' and account_id:
+        try:
+            transactions = load_data(
+                f'https://api.hackerone.com/v1/programs/{account_id}/billing/transactions',
+                api_token,
+                api_identifier,
+                params=None,
+            )
+        except requests.exceptions.HTTPError as e:
+            log.error(e)
+        else:
+            db.insert(
+                landing_table,
+                dryrun=dryrun,
+                values=[
+                    {
+                        'timestamp': timestamp,
+                        'type': transaction.get('type', ''),
+                        'activity_date': get_path(
+                            transaction, 'attributes.activity_date'
+                        ),
+                        'activity_description': get_path(
+                            transaction, 'attributes.activity_description'
+                        ),
+                        'bounty_award': get_path(
+                            transaction, 'attributes.bounty_award'
+                        ),
+                        'bounty_fee': get_path(transaction, 'attributes.bounty_fee'),
+                        'debit_or_credit_amount': get_path(
+                            transaction, 'attributes.debit_or_credit_amount'
+                        ),
+                        'balance': get_path(transaction, 'attributes.balance'),
+                        'id': get_path(transaction, 'relationships.report.data.id'),
+                        'url': get_path(transaction, 'relationships.report.links.self'),
+                    }
+                    for transaction in transactions
+                ],
+            )
+    else:
+        try:
+            reports = load_data(
+                'https://api.hackerone.com/v1/reports',
+                api_token,
+                api_identifier,
+                params={'filter[program][]': program_name},
+            )
+        except requests.exceptions.HTTPError as e:
+            log.error(e)
+        else:
+            db.insert(
+                landing_table,
+                dryrun=dryrun,
+                values=[
+                    {
+                        'timestamp': timestamp,
+                        'id': report.get('id', ''),
+                        'type': report.get('type', ''),
+                        'title': report['attributes'].get('title', '')
+                        if 'attributes' in report
+                        else '',
+                        'state': report['attributes'].get('state', '')
+                        if 'attributes' in report
+                        else '',
+                        'created_at': report['attributes'].get('created_at', '')
```

Comment:
this is a `timestamp_ltz` above so I think setting it to `''` will raise an error